server/models/Group.js

```diff
@@ -15,6 +15,7 @@ const GroupSchema = new Schema({
         type: Schema.Types.ObjectId, 
         ref: 'User'
     }],
+    image: {type: String},
```

Comment:
should we have a default group image hosted url here? or should the user be required to upload a group picture on creation?

Reply:
Good point, I'm thinking we could include a default group image client side that we use if the image url doesnt exist. One benefit of doing that would be less bandwidth usage on Cloudinary. We probably won't surpass the free tier limit but wouldn't hurt to limit it. What do you think?

Reply:
Yes that makes even more sense and less requests the better to an external api! What do you think would be a good image? I feel like it needs to still look good in the group card grid but also maybe should be evident the image is not representative of the group since its a default one?

Reply:
How about this one?
![ad6c2fcc62ecc2eb14182d8bc43bcd266071bf88](https://user-images.githubusercontent.com/45152761/106274768-c507a000-6202-11eb-8c8a-27f29773aa93.png)

Reply:
I'm good with this choice.